public/static/local/transcribe/homepage.md

```
@@ -1,20 +1,18 @@
-    ![aircraft](/static/local/aviation/airplane1_hmm0001_48.jpg "Aircraft")
-- ### [Airports](/search?q=airport*)
-    ![airports](/static/local/aviation/airport_asm0299_678.jpg "Airports")
-- ### [Advertising](/search?q=advertising+OR+advertisement*+OR+poster*+OR+brochure*+OR+promotional)
-    ![ads](/static/local/aviation/Ads4_DukeD.jpg "Advertising")
-- ### [Pan American World Airways](/search?q=+%22pan+american+world%22+OR+%22pan+american+airways%22+OR+%22pan+am%22)
-    ![panam](/static/local/aviation/Pan_Am1_hmm0001_301.jpg "Pan American World Airways")
-- ### [Pilots and Flight Attendants](/search?q=pilot+OR+pilots+OR+stewardess+OR+stewardesses+OR+steward+OR+%22flight+attendant%22+OR+%22flight+attendants%22)
-    ![pilots](/static/local/aviation/flight_attendant1_asm0341_67067.jpg "Pilots and Flight Attendants")
-- ### [Early Commercial Aviation](/search?before=1945) 
-    ![early](/static/local/aviation/early_aviation1_asm0341_126581.jpg "Early Commercial Aviation")
+- ### [Untranscribed](/search?transcription_status=untranscribed)
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)